Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -367,7 +367,7 @@ open class CarPlayMapViewController: UIViewController {
             cameraOptions.pitch = 0
             navigationMapView.mapView.mapboxMap.setCamera(to: cameraOptions)
             
-            navigationMapView.fitCamera(to: [activeRoute])
+            navigationMapView.fitCamera(to: routes)
```

Comment:
It should have the same camera behavior as the CarPlay preview routes, that all the routes should be included.